### ISSUE TITLE
Gaussian Beam vs MonoChromatic Field with Circular Aperture

### ISSUE THREAD

**Issue by MarnixMeersman** [rafael-fuente/diffractsim#49]
In real life setups, we try to reach a gaussian beam by having a very small pinhole right behind the laser to approximate a gaussian beam distribution. 

I see that for the use of a gaussian beam, we first need to define a wavefield (mono/poly chromatic etc), then add gaussion with a waist diameter specified. 

What is happening under the hood here? Is it the same as adding a circular aperture of the same width? 

Just wondering. Thanks

**Comment by hadmack** [comment]
Yes, the GaussianBeam works the same was as a circular aperture. It's just a plane wave with a Gaussian apodization function applied instead, which is correct E field for the waist of a Gaussian beam.

Most lasers emit an approximately Gaussian beam with various levels of asymmetry and higher order mode content. Putting a pinhole in front of the laser is typically done to approximate a point source _spherical wave_. The circular apodization of the pinhole really diffracts to form an Airy disk in the far field, but the central lobe is a pretty uniform spherical wavefront.
